tests/virt_autotest/virsh_internal_snapshot.pm

```diff
@@ -26,6 +25,11 @@ sub run_test {
             record_info "Skip internal snapshot on $guest", "SEV/SEV-ES guest $guest does not support internal snapshot";
             next;
         }
+        if (script_run("virsh dumpxml $guest | grep loader |grep pflash") == 0) {
```

Comment:
If you want to find value of specific element in xml file, I think ```xmlstarlet sel``` is much better, because it just check the specified element and value instead a global search which tends to be not very reliable sometimes.

Reply:
Good point. Will update it.